docs/installation-configuration.md

```diff
@@ -96,6 +96,43 @@ Example of overriding the `runtimeArtifact` parameter with plugin configuration:
 </plugin>
 ```
 
+#### Calculating the runtime artifact version
+
+If a version is provided in the `runtimeArtifact` parameter configuration it will be used as the version of the runtime artifact used for the Liberty installation unless it is overridden by the `libertyRuntimeVersion` property.
+
+If a version is not provided with the `runtimeArtifact` parameter configuration or `libertyRuntimeVersion` property then, first, the runtime artifact groupId and artifactId will be calculated (via parameter config, properties, or default values).   If this runtime artifact groupId and artifactId matches those of a project dependency then the version of this dependency will be set as the version of the runtime artifact used for Liberty installation.   If there is no matching project dependency than the project's dependencyManagement is checked for a dependency matching the runtime artifact groupId and artifactId and if such a dependency is found, this version will be set as the version of the runtime artifact used for Liberty installation.
+
+If a version is not provided with the `runtimeArtifact` parameter configuration or `libertyRuntimeVersion` property and also there is no matching dependency listed as a project dependency or in the project's dependency management, then a default version value is chosen, generally the most recent available version of Liberty.
```

Comment:
These three paragraphs are pretty wordy. I think maybe some sort of numbered list that details the decision tree would be easier to follow. I'll try to write it up later today.

Reply:
Yeah a lot of repetition.. was thinking some indenting or numbering was needed but wanted to at least get it out there while I had an idea of what it was supposed to do.

Reply:
> The code changes make sense. Just had a couple comments on the doc.

Thanks for taking a look !